AdaptivePELE/simulation/simulationrunner.py

```diff
@@ -9,6 +9,7 @@
 import numpy as np
 import ast
 import glob
+import subprocess32
```

Comment:
This should be an optional dependency, since it is not part of the standard library of python and is not always needed, see example of importing sklearn in lines 18-22 in this file